OpenRA.Platforms.Default/Sdl2Input.cs

```diff
@@ -20,6 +20,7 @@ class Sdl2Input
 		MouseButton lastButtonBits = (MouseButton)0;
 
 		public string GetClipboardText() { return SDL.SDL_GetClipboardText(); }
+		public int SetClipboardText(string text) { return SDL.SDL_SetClipboardText(text); }
```

Comment:
Please change this to return a bool: `SDL.SDL_SetClipboardText(text) == 0`.  The "non-zero integer means an error" status code isn't conventional in C#.